### ISSUE TITLE
[BUG] [Linter] False positives for duplicate attributes H037

### ISSUE THREAD

**Issue by kai687** [Riverside-Healthcare/djLint#633]
- [X] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [X] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [X] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS (whatever the latest is)
 - Python Version 3.11.3
 - djLint Version 1.27.0
 - template language: HTML

## Issue

The rule H037 "Duplicate attribute found" flags attributes as duplicate even if they only partially match, so SVG attributes "width" and "stroke-width" are flagged. 

## How To Reproduce

1. Create a file `test.html` with the content: 
   
   ```html
   <svg width="16" stroke-width="2"></svg>
   ```

2. Run `djlint test.html`

## Contents of .djlintrc/pyproject.toml [tool.djlint]

No config.
